ui/src/components/SelectImportPage/index.tsx

```diff
@@ -251,7 +254,7 @@ export const SelectImportPage = () => {
           handleChangeGroup={handleChangeGroup}
           handleSearchValue={handleSearchValue}
           locationGroupId={locationGroupId}
-          componentTypesResult={componentTypesResult}
+          componentTypesResult={importableComponentTypes}
```

Comment:
nit: componentTypesResult (as a prop) could now be named as `importableComponentTypes` or such, since it doesn't represent the full retrieved list, but theoretically the subset that can be imported. It would also apply to a couple of places (since all import-related selections now have this constraint).

However, I understand that naming change would modify the interface and cascade through components.

Reply:
That's a good call! I'll make that change to make it clearer what is expected to be passed down here :)